email-templates/danish_templates/dinero_post_first_sale.xml

```diff
@@ -0,0 +1,349 @@
+<mjml>
+  <mj-head>
+      <mj-title>Dit første fakturasalg</mj-title>
+          <mj-font name="Montserrat" href="https://fonts.googleapis.com/css?family=Montserrat:400,600"/>
+          <mj-font name="Roboto" href="https://fonts.googleapis.com/css?family=Roboto:400,700"/>
+          <mj-breakpoint width="480px" />
+              <mj-attributes>
+               <mj-section padding="64px 85px" background-color="#ffffff"/>
+              <mj-wrapper padding="0"/>
+               <mj-text line-height="20px"/>
+              </mj-attributes>
+
+                  <!--Divider-->
+                  <mj-style>
+                  .border-top {
+                    border-top: 3px solid #E4F0F7;
+                   }
+                  </mj-style>
+                  <!--padding on mobile-->
+                  <mj-style>
+                  @media only screen and (max-width:480px) {
+                  .section td {
+                  padding-left:10px!important;
+                  padding-right:10px!important;
+                    }
+                  }
+                  </mj-style>
+                  <!--Hide on mobile, use css-class="hide_on_mobile"-->
+                  <mj-style>
+                  @media only screen and (max-width: 480px){ *[class~=hide_on_mobile] { display: none !important;} }
+                  </mj-style>
+                  <!--Removing underscore on links-->
+                  <mj-style inline="inline">
+                  .link-nostyle { color: #0000FF; text-decoration: none }
+                    </mj-style>
+
+  <mj-preview>Du har modtaget din første tidlige betaling</mj-preview>
+
+  </mj-head>
+  <mj-body background-color="#E4F0F7">
+
+  <mj-raw><!-- Company Header Tables --></mj-raw>
+      <mj-section padding="0">
+          <mj-column>
+              <mj-table padding="21px 0">
+                  <tr>
+                      <td style="padding:0 0 0 18px; line-height:1;">
+                          <img width="111"
```

Comment:
I do not see the moneyflow logo in the Figma file. This could be important not to show. Anna would know.

Reply:
Comment should be on `dinero_invoice_payer_email`. Marking this comment resolved